src/process_model_output.R

```diff
@@ -51,14 +52,14 @@ find_quantile_group <- function(value, breaks, labels) {
 
 percentile_categories <- c("very low", "low", "average", "high", "very high")
 values_categorized <- total_storage_data %>%
-  left_join(quantile_df, by = c("hruid", "DOY")) %>%
```

Comment:
FYI, I removed this by "DOY" because it throws ```Error: `by` can't contain join column `DOY` which is missing from LHS``` - when I check total_storage_data, it doesn't have DOY

Reply:
`DOY` must have been lost earlier then?  I think that will be needed though, otherwise the join could mismatch days.